### PR TITLE
copyedit(text2vec-openai): fix schema example, update OpenAI links

### PR DESCRIPTION
**Pull request by dandv**
### What's being changed:

* fix schema example
* update OpenAI links (no longer in beta)
+ rm dupe vectorizer property from schema
+ fix model babbage -> ada
+ rm '.md' from links
+ more semantic links
+ uniform heading capitalization
+ rm EOL spacing and nonstandard spaces

### Type of change:

- [x] Documentation updates (non-breaking change which updates documents)

### How Has This Been Tested?

`npm start` on locahost

### CODE REVIEW

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md [hunk 1]
```diff
@@ -49,14 +51,14 @@ services:
 ...
 ```
 
-* Note: you can also use the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
```
Comment (sebawita) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md:52]: The link should include the `.md` in the path, as this is used by docusaurus to create a path

Comment (sebawita) [reply developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md:52]: Can we add `.md` back to the other links as well?

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md [hunk 2]
```diff
@@ -49,14 +51,14 @@ services:
...
+* Note: you can also use the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose#configurator) to create a Weaviate setup with this module.
```
Comment (sebawita) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md:54]: Could we ase the :::note admonition for this?

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md (hunk 1 above)
Comment (dandv) [reply developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md:52]: I can re-add the `.md`, and also want to understand what exactly the rule is for when `.md` needs to be present at the end of links.

I've tested locally every link that I had removed `.md` from, and clicking did go to the correct link. I've also tested the same links [on the Netlify build](https://63e9f1cbfb18ad3c3f5c4dda--tangerine-buttercream-20c32f.netlify.app/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai#how-to-configure) and they worked as well.

Does Docusaurus fail to create the links without the trailing `.md` on the live site?

Doesn't seem to be the case - e.g. at https://weaviate.io/google-summer/gsod-22#users-and-contributors, the link in "people who [make their own Weaviate module](https://weaviate.io/developers/contributor-guide/weaviate-modules/how-to-build-a-new-module)" doesn't have the trailing `.md` in the source (https://github.com/weaviate/weaviate-io/edit/main/src/pages/google-summer/gsod-22.md, line 31), and the link target *is* a file - https://github.com/weaviate/weaviate-io/blob/main/developers/contributor-guide/weaviate-modules/how-to-build-a-new-module.md

Comment (databyjp) [reply developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai.md:52]: Hey @dandv - afaik it is preferable to use .md in the links as they are more robust - see details in [this doc](https://docusaurus.io/docs/markdown-features/links). 

But sometimes we can't use `.md` in links - for example when we componentize passages of text. So in those cases we have had to use relative URLs. 

(It's also possible that they're mistakes 😅).

**Pull request merged**